src/platforms/javascript/guides/capacitor/troubleshooting.mdx

```diff
@@ -5,6 +5,31 @@ keywords: ["minimum version", "deployment target"]
 sidebar_order: 1000
 ---
 
+## Different Versions for Sibling SDKs 
+
+If you see the following error during build time, the sibling SDKs installed in your app have mismatched versions:
+
+```js
+  node_modules/@sentry/types/types/globals.d.ts:2:11 - error TS2451: Cannot redeclare block-scoped variable '__DEBUG_BUILD__'.
+```
+
+To avoid any issues when installing packages or using the code on another machine, you must use the same version of Sentry Capacitor and @Sentry/Angular (or any other sibling SDK) that is referred by the peer dependency of Sentry Capacitor in your app.  
```

Comment:
Not sure if the word "referred" is correct here? 
```suggestion
To avoid any issues when installing packages or using the code on another machine, you must use the same version of Sentry Capacitor and @Sentry/Angular (or any other sibling SDK) that is referenced by the peer dependency of Sentry Capacitor in your app.  
```

Reply:
looks good for me, thanks for the review :D